mesonbuild/interpreter/type_checking.py

```diff
@@ -893,15 +893,21 @@ def _pkgconfig_define_convertor(x: T.List[str]) -> PkgConfigDefineType:
     convertor=_pkgconfig_define_convertor,
 )
 
+INCLUDE_TYPE = KwargInfo(
+    'include_type',
+    str,
+    default='preserve',
+    since='0.52.0',
+    validator=in_set_validator({'system', 'non-system', 'preserve'})
+)
```

Comment:
This particular file's changes could be done in the previous patch, "interpreter: port dependency include_type to typed_kargs".